docs/cpp/cmake-quickstart.md

```diff
@@ -0,0 +1,73 @@
+---
+Order: 9
+Area: cpp
+TOCTitle: CMake Quick Start
+ContentId: 55b5d15c-a020-4808-941f-e0255751a5f7
+PageTitle: Create a CMake project with the CMake Quick Start
+DateApproved: 5/29/2024
+MetaDescription: Create a Hello World project by using the CMake Quick Start in the CMake Tools Visual Studio Code extension
+---
+# Create a CMake hello world project with CMake Quick Start
+
+In this article, you'll learn how to create a CMake hello world project from scratch using the CMake Tools extension in VS Code.
+
+If you have an existing CMake project that already has a `CMakeLists.txt` file in the root directory but no CMake presets, you can skip to [Create a `CMakePresets.json` file](#Create-a-CMakePresets.json-file) to configure your project with CMake presets.
+
+Otherwise, create a folder for a new project. From the Terminal window, create an empty folder called `HelloWorld`, navigate into it, and open VS Code in that folder by entering the following commands:
+
+```bash
+mkdir helloworld
+cd helloworld
+code .
+```
+The ```code .``` command opens VS Code in the current working folder, which becomes your "workspace".
```

Comment:
Maybe this should be a single-backtick codespan? Not sure how it's rendered on the website.

Reply:
Not sure why I suggested that.. Should indeed be single-backtick!